lms/templates/student_account/account_settings.html

```diff
@@ -77,6 +77,11 @@
 <script type="text/javascript">
      window.auth = ${ auth | n, dump_js_escaped_json };
      window.isActive = ${ user.is_active | n, dump_js_escaped_json };
+     window.additionalSiteSpecificDeletionText = "${ static.get_value('SITE_SPECIFIC_DELETION_TEXT', 'MIT Open Learning, Wharton Executive Education, and Harvard Medical School.') | n, js_escaped_string }";
```

Comment:
Could you tag the string `'MIT Open Learning...` with the `_()` so that it's marked for translation?

Reply:
I think, that I can. Let me test that.

Reply:
It worked!